src/main/java/org/casbin/casdoor/config/CasdoorAutoConfigure.java

```diff
@@ -0,0 +1,28 @@
+package org.casbin.casdoor.config;
+
+import org.casbin.casdoor.Casdoor;
+import org.casbin.casdoor.CasdoorException;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import javax.annotation.Resource;
+
+/**
+ * @author tangyang9464
+ */
+@Configuration
+@EnableConfigurationProperties(CasdoorConfiguration.class)
+public class CasdoorAutoConfigure {
+    @Resource
+    private CasdoorConfiguration properties;
```

Comment:
For mandatory dependencies, it is recommended to use constructor injection.

Reply:
changed